File: web/app/model/pam/Committee.js
Ext.define('iFlat.model.pam.Committee', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'committee.id', mapping: 'id', type: 'string'},
        {name: 'committee.pbName', mapping: 'pbName', type: 'string'},
        {name: 'committee.electionTime', mapping: 'electionTime', type: 'date'},
        {name: 'committee.people', mapping: 'people', type: 'number'},
    ]
});

File: src/main/java/com/iflat/pam/bean/Committee.java
package com.iflat.pam.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/7/29.
 */
public class Committee {

    private String id;
    private String pbName;
    private Date electionTime;
    private int people;

    public int getPeople() {
        return people;
    }

    public void setPeople(int people) {
        this.people = people;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPbName() {
        return pbName;
    }

    public void setPbName(String pbName) {
        this.pbName = pbName;
    }

    public Date getElectionTime() {
        return electionTime;
    }

    public void setElectionTime(Date electionTime) {
        this.electionTime = electionTime;
    }
}
